node/src/components/storage.rs

```diff
@@ -371,6 +373,12 @@ pub(crate) struct Storage {
     switch_block_era_id_index: BTreeMap<EraId, BlockHash>,
     /// A map of deploy hashes to hashes of blocks containing them.
     deploy_hash_index: BTreeMap<DeployHash, BlockHash>,
+    /// Whether or not memory deduplication is enabled.
+    enable_mem_deduplication: bool,
```

Comment:
Won't we always want to enable it? Or is this temporary, so we can compare performance with and without the deduplication?